### DISCUSSION TITLE
Network v14 'Chocolate'

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#320] [Network + Implementations Announcements]
This is for discussing about the upcoming [Network v14 'Chocolate'](https://github.com/filecoin-project/community/discussions/74#discussioncomment-1398542) 🍫 !

**Comment by arajasek** [comment]
For anyone wondering, it's called `Chocolate` because this upgrade has lots of goodies for the network -- small UX and protocol improvements that the network doesn't _need_, but really _wants_!

**Comment by Angymr** [comment]
https://github.com/filecoin-project/community/discussions/320#discussioncomment-1398669
